pandas/core/dtypes/dtypes.py

```diff
@@ -1232,3 +1233,91 @@ def __from_arrow__(
             results.append(iarr)
 
         return IntervalArray._concat_same_type(results)
+
+
+@register_extension_dtype
+class DateDtype(PandasExtensionDtype):
+    """
+    An ExtensionDtype to hold a single date.
+    The attributes name & type are set when subclasses are created.
+    """
+
+    _date_aliases = {"date", "date64"}
+    _unit = "D"
+    _numpy_dtype = np.dtype("datetime64[D]")
+
+    def __str__(self):
+        return "date"
+
+    @property
+    def name(self) -> str_type:
+        """
+        The alias for DateDtype is ``'string'``.
+        """
+        return str(self)
+
+    @property
+    def type(self):
+        return Timestamp
+
+    @property
+    def na_value(self):
+        return NaT
```

Comment:
This needs to be discussed in detail. We need to decide if we want `NaT` or `NA` semantics. I think I'd prefer NA.

Reply:
-1 on changing to NA at this time, we have a very well established policy of NaT in *all* datetimelikes.